approach.tex

```diff
@@ -1,9 +1,9 @@
 \section{Approach}
 \label{approach}
 The new \texttt{vtkGPUVolumeRayCastMapper} uses a ray casting
-technique~\citep{engel_real-time_2006} for volume rendering which is a
-state-of-the-art for volume rendering on modern graphics platforms.
-Algorithmically, at a high level, it is similar to the older version of this
+technique ~\citep{engel_real-time_2006} for volume rendering which
```

Comment:
@aashish24 You should not be adding a space before the `~` character. Refer to https://github.com/aashish24/VTKVR/pull/28#discussion_r118722773 for a detailed explanation.